pkg/domain/infra/abi/containers.go

```diff
@@ -709,3 +712,48 @@ func (ic *ContainerEngine) ContainerRun(ctx context.Context, opts entities.Conta
 	}
 	return &report, nil
 }
+
+func (ic *ContainerEngine) ContainerLogs(ctx context.Context, containers []string, options entities.ContainerLogsOptions) error {
+	if options.Writer == nil {
```

Comment:
imho this should be caught at the client level i.e. cmd/...

Reply:
We're setting that always in `cmd` but I wanted to be defensive in case things change in the future.